[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:36:42Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
config-server           active      1  mongodb-k8s  6/edge    54  10.152.183.145  no       
mongos-k8s              active      1  mongos-k8s              0  10.152.183.86   no       
shard0                  active      1  mongodb-k8s  6/edge    54  10.152.183.170  no       Primary

Unit              Workload  Agent  Address      Ports  Message
config-server/0*  active    idle   10.1.57.141         
mongos-k8s/0*     active    idle   10.1.57.136         
shard0/0*         active    idle   10.1.57.142         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:30:26 ERROR unit.config-server/0.juju-log config-server:4: Failed to add shard shard0 to the config server, error=OperationFailure('Authentication failed., full error: {\'ok\': 0.0, \'errmsg\': \'Authentication failed.\', \'code\': 18, \'codeName\': \'AuthenticationFailed\', \'$clusterTime\': {\'clusterTime\': Timestamp(1726623026, 1), \'signature\': {\'hash\': b\'\\xb1?>:a\\x11\\x00k@\\x16P"\\x98\\xd9O\\xc7x\\xfb\\x08\\xe1\', \'keyId\': 7415789287456636952}}, \'operationTime\': Timestamp(1726623026, 1)}')
unit-config-server-0: 01:30:26 ERROR unit.config-server/0.juju-log config-server:4: shard0 shard does not have the same auth as the config server.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...